packages/ra-core/src/core/useConfigureAdminRouterFromChildren.tsx

```diff
@@ -79,7 +79,6 @@ const useRoutesAndResourcesFromChildren = (
     // We need to know right away wether some resources were declared to correctly
     // initialize the status at the next stop
     const doLogout = useLogout();
-    const { authenticated } = useAuthState();
```

Comment:
This is super weird. Could you do some archeology in the git history to understand why it was put here in the first place?

Reply:
I think I got it. This has been done to ensure we re-evaluate permissions when users sign in with different accounts. But I believe we have better ways to do that in the `usePermissions` hook that should re-execute on user change

Reply:
Added a test to validate permissions are refreshed after users log out and back in with different permissions